src/openzaak/config/templates/admin/config_detail.html

```diff
@@ -0,0 +1,75 @@
+{% extends "admin/base_site.html" %}
+{% load i18n config %}
+
+{% block title %} {% trans "Service configuratie" %} {{ block.super }} {% endblock %}
+
+{% block breadcrumbs %}
+<div class="breadcrumbs">
+<a href="{% url 'admin:index' %}">{% trans 'Home' %}</a>
+&rsaquo; {% trans 'Service configuratie' %}
+</div>
+{% endblock %}
+
+{% block content %}
+    <h1>{% trans 'Open Zaak API dashboard' %}</h1>
+    <div id="content-main">
+        <!-- NLX config -->
+        <section class="module">
+            <table class="table">
+                <caption>NLX</caption>
+                <thead>
+                    <tr>
+                        <th scope="col">Parameter</th>
+                        <th scope="col">Waarde</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><strong>NLX directory</strong></td>
+                        <td>{{ nlx.directory_url| urlize }}</td>
+                    </tr>
+                    <tr>
+                        <td><strong>Outway address</strong></td>
+                        <td>{{ nlx.outway| urlize }}</td>
+                    </tr>
+                </tbody>
+            </table>
+        </section>
```

Comment:
Add change button "Wijzigen" for this section (non-wizard)

Reply:
the button is added